summaries/sprint1.md

```diff
@@ -2,6 +2,7 @@
 
 ## Ahmed
 
```

Comment:
@ahmedc3ca we can remove this file right?

Reply:
We should keep all the sprint briefs no?

Reply:
I meant remove it from the PR no?